frontend/src/store/FamLoginUserState.ts

```diff
@@ -103,7 +103,9 @@ const getApplicationsUserAdministers = () => {
                 });
             }
         });
-        applicationList.sort((first, second) => first.id - second.id);
+        applicationList.sort((first, second) => {
+            return first.description < second.description ? -1 : 1
+        });
         if (famApp) applicationList.unshift(famApp); // add FAM to the first if FAM Admin.
```

Comment:
this condition was added to put FAM at the top of the list intentionally, should it be removed?

Reply:
Hi Jota, I think we can keep it, cause only us will see that anyway

Reply:
sure, thanks!